test/unit/util_string.js

```diff
@@ -63,3 +63,14 @@ test('unusedName', t => {
     );
     t.end();
 });
+
+test('stringify', t => {
```

Comment:
Could you also add a test that the new stringify function handles nested objects correctly?

E.g. `{a: Infinity, b: {b1: Infinity, b2: 42}}`